libs/server/src/lib/api/detailed-status-calculator.ts

```diff
@@ -527,3 +549,13 @@ function collectPreviousErrorsFrom(idx: number, lines: string[]): string[] {
 
   return xs;
 }
+
+function parseTime(s: string) {
+  if (s.endsWith('ms')) {
```

Comment:
What happens otherwise? Throw an error? Log an error? Something?

Reply:
Just return the string that comes out put `ng build` or `ng serve`. I just want to format it to a nice string like  `1.02s`, but it's fine if it remains ugly as well. :)